cognite/neat/rules/models/information/_rules.py

```diff
@@ -99,6 +99,8 @@ class InformationMetadata(BaseMetadata):
             "typically information architects are considered as contributors."
         ),
     )
+    license: str | None = None
+    rights: str | None = None
```

Comment:
These are in the tutorial but actually not read by the information rules, I added them here as we should at least read them. Alternative is to remove them from the tutorial.

Reply:
Great !